tst/testinstall/perm.tst

```diff
@@ -314,12 +314,38 @@ gap> PermList(Concatenation([1..70000],[70001,1,70003]));
 fail
 
 #
-# LARGEST_MOVED_POINT_PERM
-#
-gap> LARGEST_MOVED_POINT_PERM((2,3));
-3
-gap> LARGEST_MOVED_POINT_PERM((2,70000));
-70000
```

Comment:
The reason for these tests was to make 100% sure we test those kernel function, and not some other installed methods.

I guess these days, I'd move those tests to `tst/testinstall/kernel/permutat.tst`